src/main/java/com/minecolonies/coremod/entity/ai/citizen/farmer/Field.java

```diff
@@ -1,13 +1,14 @@
 package com.minecolonies.coremod.entity.ai.citizen.farmer;
 
 import com.minecolonies.api.colony.permissions.Action;
+import com.minecolonies.api.util.constant.Constants;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.constant.Constants'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)